content/doc/book/managing/system-properties.adoc

```diff
@@ -1319,6 +1319,27 @@ properties:
     Development-mode only: Set to `true` to not skip showing the setup wizard during Jenkins development.
     This property is only effective the first time you run Jenkins in given `JENKINS_HOME`.
 
+- name: jenkins.install.SetupWizard.adminInitialApiToken
+  tags:
+  - security
+  - packaging
+  def: undefined
```

Comment:
Wouldn't it be clearer to the reader if the last sentence were included as the explanation of the default behavior?  That would make the default value clear to readers and follow the same pattern used by [executable-war](https://www.jenkins.io/doc/book/managing/system-properties/#executable-war).
```suggestion
  def: The default admin account will not have an API Token unless a value is provided for this system property
```